### ISSUE TITLE
feat: ✨ Theme for the CLI create sub-command

### ISSUE THREAD

**Issue by philippart-s** [slidesk/slidesk#161]
The main idea is to add an option (`-t`, `--theme`) to the CLI `create` sub-command.
This option will allow to specify which template to use.
Two possibilities:
 - a _core_ template (i.e packaging and hosting by SliDesk team): `slidesk create -t cool-template`
 - a community template (i.e hosted on GH): `slidesk create -t https://github.com/philippart-s/cool-sd-theme`

This command will create a more sophisticated tree structure with some pre-configured things:
 - plugins
 - css
 - images
 - slides
 - js
 - ... 

In a nutshell this will replace the default SliDesk files (`main.sdf` and `custom.css`).

**Comment by gouz** [comment]
will

**Comment by gouz** [comment]
2.8.0 ;)
